scripts/RunXGBoostNativeHadoopREADME.txt

```diff
@@ -0,0 +1,70 @@
+This file shows all steps need to run xgboost off Hadoop on mr-0xd2.  All AWS keys and stuff have been set up on this machine.  In addition, I put the data in the following place: s3://h2o-public-test-data/xgboost-dist/.  Do a s3cmd ls s3://h2o-public-test-data/xgboost-dist/ to see what directories and files are there.  You will notice that there are airlines and higgs directories containing airlines and higgs dataset in the correct format for xgboost in ingest.  
```

Comment:
I think it would be very useful to document how these datasets were produced.

Reply:
Honza:

I only have the higgs and airline dataset conversion and feel that once you wrote your general dataset conversion, that will be more useful than what I have here.  Let me know what you think.  I can easily added those two scripts I sent you here.